pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.sonatype.buildsupport</groupId>
     <artifactId>public-parent</artifactId>
-    <version>37</version>
+    <version>44</version>
```

Comment:
`44` is the minimum.  If possible it would be good to bump this all the way to the latest, which is `55`